src/components/Footer/index.sass

```diff
@@ -0,0 +1,48 @@
+.contact-box
```

Comment:
Also while editting we can use rem everywhere, and may be add rem default as 10px in the parent css.
I can do that if you miss